sdv/lite/tabular.py

```diff
@@ -43,10 +46,21 @@ def __init__(self, name=None, metadata=None):
                           'detected from your data. This process may not be accurate. '
                           'We recommend writing metadata to ensure correct data handling.')
 
+        if metadata is not None and constraints is not None:
+            if isinstance(metadata, Table):
+                metadata = metadata.to_dict()
+
+            metadata['constraints'] = []
+            for constraint in constraints:
+                metadata['constraints'].append(constraint.to_dict())
+
+            constraints = None
+
         if name == SPEED_PRESET:
             self._model = GaussianCopula(
                 table_metadata=metadata,
-                categorical_transformer='label_encoding',
+                constraints=constraints,
+                categorical_transformer='categorical_fuzzy',
```

Comment:
why are we changing this?

Reply:
@npatki mentioned that for GaussianCopula, it expects continuous values, and that label encoding and categorical are not continuous.